README.md

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
```

Comment:
I did not asked to add the task description. I asked you to  **add README where should be an explanation of what does the application and how looks the result.**